runtime/log.go

```diff
@@ -0,0 +1,26 @@
+package runtime
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+)
+
+// Create a logger that can be passed around through the environment.
```

Comment:
Good call on changing this documentation by the way, still getting used to how things should be documented.